modules/reactor/src/meshgenerators/PatternedPolygonPeripheralModifierBase.C

```diff
@@ -235,6 +269,10 @@ PatternedPolygonPeripheralModifierBase::generate()
                                                                    _type,
                                                                    _name);
 
+    // Convert the linear mesh to quadratic mesh
+    if (order == 2)
+      mesh->all_second_order();
```

Comment:
let's keep track of quad8 or 9, then use all_second_order for quad8 and all_complete_order (if it exists) for quad9